packages/web-react/src/components/Breadcrumbs/__tests__/Breadcrumbs.test.tsx

```diff
@@ -1,4 +1,6 @@
 import '@testing-library/jest-dom';
+import { render, prettyDOM } from '@testing-library/react';
```

Comment:
probably forgotten import of `prettyDOM`?

Reply:
Yes, fixed. Thanks.